examples/src/utils.js

```diff
@@ -0,0 +1,327 @@
+import { VASTClient, VASTTracker } from '@dailymotion/vast-client';
+const mainContent =
+  'https://dmplayer.storage.googleapis.com/tech_test/midnight_sun_720p.mp4';
+const videoPlayer = document.getElementById('myVideo');
+const variationImg = document.getElementById('variationImg');
+const variationContainer = document.getElementById('variationContainer');
+
+//vastTracker for linear ads
+let vastTracker = null;
+//vastTracker for companion ads
+let companionVastTracker = null;
+
+/**
+ * Get the first valid creative from an ad
+ */
+export const getCreative = (ad) => {
+  if (!ad.creatives) {
+    return null;
+  }
+  const validCreative = ad.creatives.find((creative) => creative.mediaFiles);
+  return validCreative || null;
+};
```

Comment:
I think it's more readable like this :) Idem for others. WDYT ?
```suggestion
export const getCreative = (ad) => {
  return ad.creatives?.find(creative => creative.mediaFiles) || null;
};
```

Reply:
yes definitly more readable !